app/services/ldap/entity.rb

```diff
@@ -59,8 +58,8 @@ def preferredLanguage
       def eduPersonAffiliation
         user.ldap_roles
             .map { |role| prefixes_for_role(role) }
-            .flatten
-            .uniq
+            .flatten!
```

Comment:
> Performance/ChainArrayAllocation: Use unchained flatten! and uniq! (followed by return array if required) instead of chaining flatten...uniq.

uyarıda parantez içinde belirtilen durum sanırım burada geçerli. Yalnızca `student` ve yalnızca `employee` rolü olan iki kullanıcı için deneme yaptığımda `eduPersonAffiliation`, `nil` dönüyor.

Reply:
Bunun testini yazmış, normalde test geçiyor ama değin gibi konsol üzerinden denediğimde nil dönüyor